bodhi/server/models/models.py

```diff
@@ -850,10 +850,12 @@ def edit(cls, request, data):
             comment += '\n\nNew build(s):\n'
             for new_build in new_builds:
                 comment += "\n- %s" % new_build
+            comment += '\n\nKarma has been reset.'
         if removed_builds:
-            comment += '\n\nRemoved build(s):\n'
+            comment += '\n\n\nRemoved build(s):\n'
             for removed_build in removed_builds:
                 comment += "\n- %s" % removed_build
+            comment += '\n\nKarma has been reset.'
```

Comment:
One more change I'd suggest - we probably don't want to say that karma was reset twice in any given operation. I suggest moving these ```comment +=``` lines into a new if statement that's outside of both of the above ifs, so that the comment only gets added once. Something similar to this would probably work:

```
if new_builds or removed_build:
    comment += '\n\nKarma has been reset.'
```

This way you don't need to edit the ```Removed build(s)``` comment either, I don't think.